resources/views/checkout/partials/sections/login/logged-out.blade.php

```diff
@@ -16,12 +16,21 @@ class="justify-center"
         </p>
 
         <template v-if="!loggedIn && (!checkoutLogin.isEmailAvailable || checkoutLogin.createAccount)">
-            <x-rapidez-ct::input.password
-                label="Password"
-                name="password"
-                v-model="checkoutLogin.password"
-                required
-            />
+            <div class="relative">
+                <x-rapidez-ct::input.password
+                    label="Password"
+                    name="password"
+                    v-model="checkoutLogin.password"
+                    required
+                />
+                <input
```

Comment:
Why do we need an absolute checkbox?

Reply:
The explanation can be found here: https://github.com/rapidez/checkout-theme/pull/121#issuecomment-2459147324